fsi-workflow/01_solidMesh/tasks.md

```diff
@@ -1,182 +1,147 @@
 
- - Mesh->FEM Mesh group
- 
-![select](./images/Groups02.png)
+- From the `Model` tav, expand the `Analysis` branch and select `FEMMeshGmsh`:
 
- - Make sure that the **BaseFeature** geometry is visible
- 
-![select](./images/Groups03.png)
- 
- - check Label in the *Tasks* tab of *Combo module** on the left
- 
-![select](./images/Groups04.png)
- 
- - click **Add** and select *root surface* (pay attention to reference frame to identify it) ->Add ->Ok
- - rename Label in Properties to **root**
+![FreeCAD: Select FEMMeshGmsh](./images/Groups01.png)
 
-![select](./images/Groups05.png)
+- Click `Mesh` -> `FEM Mesh group`:
 
- - repeat for other surfaces (they are **4**: upper, lower, tip and trailing. pay attention to trailing edge surface)
- - rename the group to **wetSurface**
+![FreeCAD: Mesh group](./images/Groups02.png)
 
- - repeat for tip surface (it will be used for single physics test)
- - rename the group to **tip**
+  In case you cannot select anything and you get an error "Active Task Dialog found!", you might need to switch to the `Tasks` tab and click `OK` or `Cancel` to exit from the previous task.
 
-Select the mesh->double click-> press apply to remesh and create groups-> ok
+- In the `Model` tab, there should now also be a `MeshGroup` under the `FEMMeshGmsh`.
 
-==IMPORTANT==: you need to remesh otherwise the groups won't be created
+![FreeCAD: Added MeshGroup](./images/Groups03.png)
 
-## Save INP file and verify
+- In the `Tasks` tab, select `Label` as `Identifier used for mesh export` and `Face, Edge, Vertex` as `Selection mode`:
 
- - keep the mesh selected
- - File->export 
- - name the file  **wing2312.inp**
- - save
- - close
- - File->Save the FreeCAD model
+![FreeCAD: Add surface to mesh group](./images/Groups04.png)
 
-open the **inp** file with a text editor:
+- Click `Add`, then click on the rendering to select the profile of the wing (pay attention to reference frame to identify it), and `OK` to add the surface to the mesh group. You can rotate the view using the cude on the upper right corner, or [using your mouse](https://wiki.freecad.org/Mouse_navigation) (e.g., by `Shift` + `right click`).
+- In the `Model` tab, select the `MeshGroup` and rename its `Label` to `root`.
 
+![FreeCAD: Rename mesh group](./images/Groups05.png)
 
- - look for **ELSET**, skip Evolume, you may remove remove all other ELSET
- - look for **NSET**, skip Nall
-     - *NSET, NSET=tip_Nodes 
-     - *NSET, NSET=wetSurface_Nodes
-     - *NSET, NSET=root_Nodes
+- Repeat for other surfaces (they are 4: `upper`, `lower`, `tip` and `trailing`. Pay attention to the trailing edge surface, you need to zoom-in to see it). Remember to first select `FEMMeshGmsh`.
+- Rename the group (**TODO: Which? How?**) to `wetSurface`
+
+- Repeat for the leading edge (it will be used for single physics tests), with group name `tip`.
+
+**TODO:** Add screenshot with final setup
```

Comment:
I think we need a screenshot here of how it should look like in the end. I got this, but I understand that it is not correct:

![Screenshot from 2024-08-02 19-56-27](https://github.com/user-attachments/assets/2771197c-f280-432b-bb01-7185fc4ed287)